components/account/update.htm

```diff
@@ -19,6 +19,14 @@
         <label for="accountPasswordConfirm">Confirm New Password</label>
         <input name="password_confirmation" type="password" class="form-control" id="accountPasswordConfirm">
     </div>
+    
+    {% if updateRequiresPassword %}
+    <div class="form-group">
+        <label for="accountPasswordOld">Current Password <small class="text-danger">* required</small></label>
+        <input name="password_old" type="password" class="form-control" id="accountPasswordOld">
+    </div>
+    
```

Comment:
@cbacon93 Please remove the extra empty line.

Reply:
Please also indent the HTML inside of the Twig conditional